pyproject.toml

```diff
@@ -1,7 +1,121 @@
 [build-system]
-requires = ["setuptools >= 40.6.0", "wheel", "setuptools_scm>=6.2"]
+requires = [
+    "setuptools >= 40.6.0",
+    "setuptools_scm>=6.2",
+]
 build-backend = "setuptools.build_meta"
 
+[project]
+authors = [
+    {name = "ESMValTool Development Team", email = "esmvaltool-dev@listserv.dfn.de"}
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Natural Language :: English",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+    "Topic :: Scientific/Engineering :: GIS",
+    "Topic :: Scientific/Engineering :: Hydrology",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+dynamic = [
+    "readme",
+    "version",
+]
+description = "A community tool for pre-processing data from Earth system models in CMIP and running analysis scripts"
+dependencies = [
+    "cartopy",
+    "cf-units",
+    "dask[array,distributed]!=2024.8.0",  # ESMValCore/issues/2503
+    "dask-jobqueue",
+    "esgf-pyclient>=0.3.1",
+    "esmf-regrid>=0.11.0",
+    "esmpy!=8.1.0",  # not on PyPI
```

Comment:
shall we remove this then?

Reply:
at least the pin, I mean, not the entire dep

Reply:
Done in 953be1c

Reply:
Bedankt :beer: